Added online mentoring tips (#1)
KansasCityWomeninTechnology/MentorPlaybook: 05-online-mentoring.md

# Online Mentoring

---

## Tips

> * Prepare like you would for in-person mentoring. Make sure you to go over the content

Author: ```suggestion
* Prepare like you would for in-person mentoring. Make sure to go over the content
```

the mentee is going to be working on.
* Introduce yourself! Send a message on the slack channel or do a quick video
meeting introducing yourself to your mentees. This is a great way to become

> familiar and comfortable around each other.

Reviewer: Love this tip! Could we also make it tech agnostic? I think that would be helpful for other events that might use other software.

* Send check in messages. It is difficult to see if someone is struggling
with their work while online mentoring, so it's a good idea to check in and
make sure everything is going smoothly.

> * Encourage screen share and video meetings. If a mentee needs help try to facilitate

Reviewer: ⛏ Less passive voice.

```suggestion
* Encourage screen share and video meetings. If a mentee needs help it's helpful to facilitate
```

> discussion through slack video meetings and screen share. Discourage the use of

Reviewer: Same here, let's take out `slack` to keep it tech agnostic.

workspace sharing extensions such as CodeTogether and LiveShare, so the mentee can
navigate the screen themselves.

> * Be accessible. Try to be available to respond quickly to any questions that arise.

Reviewer: ```suggestion
* Be accessible. Work to be available to respond quickly to any questions that arise.
```
⛏ Or `Plan to`, or just `Be available`. Again, let's make this less passive.

It is perfectly fine to step away from your computer and take a break,

> but make sure to inform your mentees if you are stepping away for a while. If you need

Reviewer: Maybe add a suggestion here to utilize whatever online status indicators are available to them?

to step away from your device for an extended period of time make sure to have another

> mentor cover for you for the time being!

Reviewer: ```suggestion
mentor cover for you!
```
⛏ A small nitpick, up to you what is clearest.

> mentor cover for you for the time being!

Reviewer: ```suggestion
mentor cover for you for the time being!
```

Think it'd be helpful to add something like this? Might be that's it's already covered and that's fine.

```Also, don't forget to encourage mentees to take a break as well!```

* Schedule sync up times. Allow sufficient time for mentees to work independently,
and schedule a time for a video meeting where all mentees can discuss their status

> and work through problems together.

Reviewer: Trying to think if this might incentivize someone to go do other things and just schedule a couple of sync up times. My thought is that being on a video call, even with the video off, would be the most collaborative and make mentors the most available/present. What do you think?

Maybe we can change this tip to say something like this:

`Allow and encourage sufficient time for mentees to work independently. Be available but also take breaks, turn off your video, or work through the material yourself when it looks like mentees have picked up some momentum. No need to hover!`

Needs some refinement, so I welcome your suggestions here.

Author: Yeah that makes sense! So I think I can add the sync up recommendation to the third bullet point and then repurpose this tip to include what you have mentioned!

Author: I'm not sure if that makes the two tips contradictory, let me know if you think it does.
